src/app/routes/registration/select-asset-dialog/select-asset-dialog.component.ts

```diff
@@ -1,31 +1,39 @@
-import { Component, OnInit } from '@angular/core';
+import {Component, Inject, OnInit} from '@angular/core';
 import { Asset } from 'iam-client-lib';
 import { ToastrService } from 'ngx-toastr';
 import { IamService } from 'src/app/shared/services/iam.service';
 import { LoadingService } from 'src/app/shared/services/loading.service';
-import { MatDialogRef } from '@angular/material/dialog';
+import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material/dialog';
```

Comment:
hmm, you are changing imports, that way we get a lot of conflicts. Maybe adding space before and after imported class would be good? As far as I see in libraries/backend we have also spaces around classes. It's easy to change in webstorm config if that's not a problem.

Reply:
corrected according to the comments above